### DISCUSSION TITLE
Add more Attribute limit

### DISCUSSION THREAD

**Discussion by Axiumyuyu** [PaperMC/Paper#11050] [Ideas]
### Is your feature request related to a problem?

In spigot.yml you can configure some Attribute limit like this
![屏幕截图 2024-07-10 120928](https://github.com/PaperMC/Paper/assets/150242870/b7d89f9a-78b9-4a58-bb1c-568598617d80)
I want to know can more Attribute limit could be added suchh as Armor, Jump Strength etc.

### Describe the solution you'd like.

Add more Attribute limit configuration

### Describe alternatives you've considered.

For Armor, when Player or some entities received a great damage, create a plugin to deal with this (compute real damage like vanilla Armor)
I do not think this is a good idea though

**Comment by Machine-Maker** [comment]
I think a potential concern with this... is that those limits aren't then synced to the client, so any client-side prediction that might use them would be incorrect leading to some desync.

**Reply by Clexus** [reply]
> I think a potential concern with this... is that those limits aren't then synced to the client, so any client-side prediction that might use them would be incorrect leading to some desync.

there should be some attributes that don't desync with the client? like attack_knockback, I think the velocity is calculated by server and then send to player. Maybe they can get added
If I'm wrong, please correct me
